cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudSubnet.java

```diff
@@ -39,6 +39,16 @@ public CloudSubnet(String id, String name, String availabilityZone, String cidr)
         this.cidr = cidr;
     }
 
+    public CloudSubnet(String id, String name, String availabilityZone, String cidr, boolean privateSubnet, boolean mapPublicIpOnLaunch) {
```

Comment:
I don't like it's called `privateSubnet` and we set the IGW from it. We request the IGW setting `boolean hasInternetGateway = awsSubnetIgwExplorer.hasInternetGatewayOfSubnet(describeRouteTablesResult, subnet.getSubnetId());` from AWS so it would make sense, that the parameter is called IGW, and we set the privateSubnet from it's value and not reversed.

Reply:
Yes, that's what I did first, but then the tests were harder to read. The two terms on aws are interchangeable. But of course, I can change.